### PR TITLE
Updated README.rst with new flows and examples

### PR DESCRIPTION
**Pull request by vroldanbet**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -103,13 +109,31 @@ Run the application against your local Cassandra:
...
 
-Requesting a new JWT (using the example credentials inserted into Cassandra above):
+Requesting a new JWT via Resource Owner Password Grant (using the example credentials inserted into Cassandra above):
 
 .. code-block:: bash
 
     $ curl --silent -X POST -u test0:test0 -d "grant_type=password&username=test0&password=test0&scope=uid" \
          "http://localhost:8080/oauth2/access_token?realm=/services" | jq .
 
+Requesting a new JWT via Implicit Flow:
```
Comment (hjacobs) [inline README.rst]: We should probably describe how to open the URL in the browser (and the consent screen step). The cURL commands are fine, but that's not what the flow usually looks like for humans :smirk:

Comment (vroldanbet) [inline README.rst]: Agreed!

**Comment by vroldanbet** [comment]
@hjacobs, updated the rst file with actual hyperlinks

File: README.rst [hunk 2]
```diff
@@ -3,16 +3,18 @@ Plan B Provider
 ===============
 
 .. image:: https://travis-ci.org/zalando/planb-provider.svg?branch=master
-    :target: https://travis-ci.org/zalando/planb-provider
+:target: https://travis-ci.org/zalando/planb-provider
```
Comment (hjacobs) [inline README.rst]: you broke the badges, please fix the indentation

Comment (vroldanbet) [inline README.rst]: @hjacobs oops, fixed that.

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**